com.sap.cloud.lm.sl.cf.process/src/main/java/com/sap/cloud/lm/sl/cf/process/message/Messages.java

```diff
@@ -61,6 +61,11 @@
     public static final String CANNOT_DETERMINE_MODULE_NAME = "Cannot determine module name";
     public static final String CANNOT_DETERMINE_CURRENT_APPLICATION = "Cannot determine current application";
     public static final String INVALID_FILE_ENTRY_NAME = "Invalid file entry name!";
+    public static final String COULD_NOT_VERIFY_ARCHIVE_SIGNATURE = "Could not verify archive signature: {0}";
+    public static final String ARCHIVE_CONTAINS_UNSIGNED_FILES = "The archive contains unsigned class files.";
+    public static final String ARCHIVE_IS_NOT_SIGNED_BY_TRUSTED_SIGNER = "The archive is not signed by a trusted signer";

```

Comment:
Change to `The achive is not signed by the trusted certificate authority \"{0}\"`.

Reply:
Done